### PR TITLE
Ids in query

### PR DESCRIPTION
**Pull request by LindsayYoung**
Will give candidate or committee if someone searches by id.

@theresaanna I wanted to make sure the 'Name' fields is populating `q=` and not using name so this provides the right functionality.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
This also makes sure you can pas in multiple fields at a time.

**Comment by LindsayYoung** [comment]
I am planning to merge @cmc333333 outstanding PRs and then remerge this.

**Comment by LindsayYoung** [comment]
this tackles 2/3s of #291

**Comment by cmc333333** [comment]
Side note: an alternative to commas is to repeat a query param, e.g.
`?myvar=val1&myvar=val2&myvar=val3` rather than `?myvar=val1,val2,val3`. Obviously a preference thing.

File: webservices/rest.py [hunk 1]
```diff
@@ -639,6 +648,18 @@ def get(self):
             args['year'] = default_year()
         year = args['year']
 
+        # Checking for special cases
+        if 'q' in args and args['q'] is not None:
+            # use committee id
```
Comment (catherinedevlin) [inline webservices/rest.py]: It wouldn't be hard to add committee_id and candidate_id to the fulltext search index, so they would automatically be found by the regular q= queries, and I think that would accomplish the same thing - would that be a better solution?

(sorry, I feel like I would have been able to suggest this before you did the work if I'd been paying better attention)

**Pull request merged**